src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -334,8 +334,7 @@ public function request($method, $uri, array $parameters = array(), array $files
      */
     protected function doRequestInProcess($request)
     {
-        // We set the TMPDIR (for Macs) and TEMP (for Windows), because on these platforms the temp directory changes based on the user.
-        $process = new PhpProcess($this->getScript($request), null, array('TMPDIR' => sys_get_temp_dir(), 'TEMP' => sys_get_temp_dir()));
+        $process = new PhpProcess($this->getScript($request), null, null);
```

Comment:
Yes, this seemed to be the reason (see 1cec45c421cc4601feb68b80b9d2424ba8e38415 and #1785).